index.d.ts

```diff
@@ -1,128 +1,130 @@
-    | 'underline'
-    | 'code'
-    | 'link'
-    | 'styled'
+    | "bold"
+    | "italic"
+    | "strike"
+    | "underline"
+    | "code"
+    | "link"
+    | "styled";
 
   type StoryblokRichtextContent = {
-    type: StoryblokRichtextContentType
+    type: StoryblokRichtextContentType;
     attrs?: {
-      level?: number
-      class?: string
-      src?: string
-      alt?: string
-      order?: number
+      level?: number;
+      class?: string;
+      src?: string;
+      alt?: string;
+      order?: number;
       body?: Array<{
-        _uid: string
-        Text: string
-        component: 'dia-tip'
-      }>
-    }
+        _uid: string;
+        Text: string;
+        component: "dia-tip";
+      }>;
+    };
     marks?: {
-      type: StoryblokRichtextMark
+      type: StoryblokRichtextMark;
       attrs?: {
-        href?: string
-        target?: string
-      }
-    }[]
-    text?: string
-    content: StoryblokRichtextContent[]
-  }
+        href?: string;
+        target?: string;
+      };
+    }[];
+    text?: string;
+    content: StoryblokRichtextContent[];
+  };
 
   type StoryblokRichtext = {
-    type: 'doc'
-    content: StoryblokRichtextContent[]
-  }
+    type: "doc";
+    content: StoryblokRichtextContent[];
+  };
 
-  export const NODE_HEADING = 'heading'
-  export const NODE_CODEBLOCK = 'code_block'
-  export const NODE_PARAGRAPH = 'paragraph'
-  export const NODE_QUOTE = 'blockquote'
-  export const NODE_OL = 'ordered_list'
-  export const NODE_UL = 'bullet_list'
-  export const NODE_LI = 'list_item'
-  export const NODE_HR = 'horizontal_rule'
-  export const NODE_BR = 'hard_break'
-  export const NODE_IMAGE = 'image'
+  export const NODE_HEADING = "heading";
+  export const NODE_CODEBLOCK = "code_block";
+  export const NODE_PARAGRAPH = "paragraph";
+  export const NODE_QUOTE = "blockquote";
+  export const NODE_OL = "ordered_list";
+  export const NODE_UL = "bullet_list";
+  export const NODE_LI = "list_item";
+  export const NODE_HR = "horizontal_rule";
+  export const NODE_BR = "hard_break";
+  export const NODE_IMAGE = "image";
 
-  export const MARK_BOLD = 'bold'
-  export const MARK_ITALIC = 'italic'
-  export const MARK_STRIKE = 'strike'
-  export const MARK_UNDERLINE = 'underline'
-  export const MARK_CODE = 'code'
-  export const MARK_LINK = 'link'
-  export const MARK_STYLED = 'styled'
+  export const MARK_BOLD = "bold";
+  export const MARK_ITALIC = "italic";
+  export const MARK_STRIKE = "strike";
+  export const MARK_UNDERLINE = "underline";
+  export const MARK_CODE = "code";
+  export const MARK_LINK = "link";
+  export const MARK_STYLED = "styled";
+
+  export type RenderOptions = {
+    blokResolvers?: {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      [key: string]: (props: any) => JSX.Element | null;
+    };
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    defaultBlokResolver?: (name: string, props: any) => JSX.Element | null;
```

Comment:
Instead of using `props: any` here you might try [`props: Record<string, unknown>`](https://43081j.com/2019/02/typescript-avoid-using-any#record-can-be-used-for-basic-objects) instead.
This has nothing to do with the original issue but might help improve the code quality and eliminate explicit `any`.

```suggestion
    blokResolvers?: {
      [key: string]: (props: Record<string, unknown>) => JSX.Element | null;
    };
    defaultBlokResolver?: (name: string, props: Record<string, unknown>) => JSX.Element | null;
```